specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-04-01-preview/WorkspaceManagerAssignments.json

```diff
@@ -0,0 +1,736 @@
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object"
+    },
+    "JobProperties": {
+      "description": "The job properties",
+      "properties": {
+        "endTime": {
+          "description": "The time the job completed",
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        },
+        "items": {
+          "description": "List of items published by the job",
+          "items": {
+            "$ref": "#/definitions/jobItem"
+          },
+          "x-ms-identifiers": [],
+          "type": "array"
+        },
+        "provisioningState": {
+          "description": "State of the job",
+          "enum": [
+            "Succeeded",
+            "InProgress",
+            "Canceled",
+            "Failed"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "provisioningState",
+            "values": [
+              {
+                "description": "The job succeeded",
+                "value": "Succeeded"
+              },
+              {
+                "description": "The job was canceled",
+                "value": "Canceled"
+              },
+              {
+                "description": "The job is in progress",
+                "value": "InProgress"
+              },
+              {
+                "description": "The job failed",
+                "value": "Failed"
+              }
+            ]
+          }
+        },
+        "startTime": {
+          "description": "The time the job started",
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        },
+        "errorMessage": {
+          "description": "Message to describe error, if an error exists",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "jobItem": {
+      "description": "Object to describe the item added",
+      "properties": {
+        "resourceId": {
+          "description": "The resource id",
+          "type": "string"
+        },
+        "status": {
+          "description": "Status of the item publication",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "InProgress"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "status",
+            "values": [
+              {
+                "description": "The item publication succeeded",
+                "value": "Succeeded"
+              },
+              {
+                "description": "The item publication failed",
+                "value": "Failed"
+              },
+              {
+                "description": "The item publication is in progress",
+                "value": "InProgress"
+              }
+            ]
+          }
+        },
+        "executionTime": {
```

Comment:
>executionTime [](http://example.com/codeflow?start=9&length=13)

consider renaming it timePublished? (I'm not a big fan of saying we execute things)